config/foreman-proxy-answers.yaml

```diff
@@ -0,0 +1,30 @@
+# Format:
+# <classname>: false - don't include this class
+# <classname>: true - include and use the defaults
+# <classname>:
+#   <param>: <value> - include and override the default(s)
+#
+# Every support plugin class is listed, so that it
+# shows up in the interactive menu
+#
+# See params.pp in each class for what options are available
+---
+foreman_proxy:
+  custom_repo: true
+puppet:
+  server: false
```

Comment:
Perhaps this should just set puppet to false by default? Not sure we should be including the agent config by default.